packages/utilities/src/memoize.test.ts

```diff
@@ -98,6 +98,101 @@ describe('memoizeFunction', () => {
     expect(func('a')).toEqual('a4');
     expect(func('a')).toEqual('a4');
   });
+
+  it('updates the cache if the cached value is null', () => {
+    let returnFalseyValue = true;
+    let falseyValue: null = null;
+    let callback = (): number | null => {
+      if (returnFalseyValue) {
+        return falseyValue;
```

Comment:
It would be clearer to just return null here instead of introducing an extra `falseyValue` variable.

Reply:
Done